test/gdb_test_program/readme.txt

```diff
@@ -0,0 +1,5 @@
+How to build gdb test program (windows only right now):
+-Install prerequisites - GNU Tools ARM Embedded 4.9 2014q4 or newer
+-Add bin directory to path - ex. "C:\Program Files (x86)\GNU Tools ARM Embedded\4.9 2015q2\bin" to path
+-Run build.bat to create gdb_test.bin and gdb_test.elf
+-Copy main.bin and main.elf into the test folder
```

Comment:
The files are in test/gdb_program_test, but generated executables (test/gdb_test.bin/elf) are in the test folder. Shall they be also in the at least same parent folder?